### PR TITLE
[DED-125] Add info about API lifecycle with dates

### PR DESCRIPTION
**Pull request by oliwiapolec**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/DED-125)

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

**Review by ale-kacperczyk — COMMENTED** [review]
The chart is really nice; I just wonder whether it's detailed enough and whether people notice these are quarters represented 🤔

File: src/pages/data-reporting/index.mdx [hunk 1]
```diff
@@ -34,12 +34,23 @@ The Reports API goes through several stages in its lifecycle.
...
-  alt="Messaging API lifecycle"
+  alt="Reporting API lifecycle"
   src="/images/messaging/livechat-api-versioning.png"
   width="800px"
   height="149px"
 />
 
+### Current versioning
+
+Refer to this chart to learn the details about our current APIs lifecycle.
```
Comment (oliwiapolec) [inline src/pages/data-reporting/index.mdx:45]:
```suggestion
Refer to this chart to learn the details about our current APIs lifecycle. One block on the chart represents a quarter of a year.
```

File: src/pages/management/index.mdx [hunk 2]
```diff
@@ -32,12 +32,23 @@ The LiveChat Configuration API goes through several stages in its lifecycle.
...
-  alt="Messaging API lifecycle"
+  alt="Configuration API lifecycle"
   src="/images/messaging/livechat-api-versioning.png"
   width="800px"
   height="149px"
 />
 
+## Current versioning
+
+Refer to this chart to learn the details about our current APIs lifecycle.
```
Comment (oliwiapolec) [inline src/pages/management/index.mdx:43]:
```suggestion
Refer to this chart to learn the details about our current APIs lifecycle. One block on the chart represents a quarter of a year.
```

File: src/pages/messaging/index.mdx [hunk 3]
```diff
@@ -16,6 +16,35 @@ As a developer, you can access our **Messaging APIs** (Agent Chat & Customer Cha
...
+<img
+  alt="Messaging API lifecycle"
+  src="/images/messaging/livechat-api-versioning.png"
+  width="800px"
+  height="149px"
+/>
+
+## Current versioning
+
+Refer to this chart to learn the details about our current APIs lifecycle.
```
Comment (oliwiapolec) [inline src/pages/messaging/index.mdx:39]:
```suggestion
Refer to this chart to learn the details about our current APIs lifecycle. One block on the chart represents a quarter of a year.
```

**Comment by oliwiapolec** [comment]
@ale-kacperczyk I added a note about the quarters to the chart description, wdyt about this?

**Comment by ale-kacperczyk** [comment]
> @ale-kacperczyk I added a note about the quarters to the chart description, wdyt about this?

I've just realized that colors used on the images don't match colors we use in the docs. 
The legacy version isn't blue but yellow, and it was intentional - we didn't want to assign colors to versions as colors should function as notifiers (yellow)/warnings (red), etc.
So... what now? 😅

**Comment by oliwiapolec** [comment]
> > @ale-kacperczyk I added a note about the quarters to the chart description, wdyt about this?
> 
> I've just realized that colors used in the images don't match colors we use in the docs. The legacy version isn't blue but yellow, and it was intentional - we didn't want to assign colors to versions as colors should function as notifiers (yellow)/warnings (red), etc. So... what now? 😅

@ale-kacperczyk yeah, that's true — I wanted to match the docs colors at first but the stable was blue, and legacy + dev-preview both yellow. To make this chart more visually appealing I decided to match the colors @bkubiak applied to his chart. Technically we could make it one color and put some descriptions on them, but then we'd need to take away the Qs representations as I think it would get too crowded :( But to be honest, for this type of charts, I think they should have some color identification so the stages are properly divided. But yeah, it's a tough one nonetheless xD

**Review by bkubiak — APPROVED** [review]

**Pull request merged**